src/core/json/jsonrpc_handler.ml

```diff
@@ -83,4 +84,17 @@ class jsonrpc_handler (id,name,params) = object(self)
 		try f() with JsonRpc_error _ -> def
 
 	method get_params = params
+
+	method get_requested_meta_list : string list =
```

Comment:
Does this really belong in here? I think this is for general JSON-RPC handling, so also eval debugging?

Reply:
Moved it to a separate class